extensions/resteasy-reactive/quarkus-resteasy-reactive-common/runtime/src/main/java/io/quarkus/resteasy/reactive/common/runtime/ResteasyReactiveConfig.java

```diff
@@ -18,6 +18,15 @@
     @ConfigItem(defaultValue = "10k")
     public MemorySize inputBufferSize;
 
+    /**
+     * The size of the output stream response buffer. If a response is larger than this and no content-length
+     * is provided then the request will be chunked.
+     *
+     * Larger values may give slight performance increases for large responses, at the expense of more memory usage.
+     */
+    @ConfigItem(defaultValue = "8191")
```

Comment:
Why not `8192`?

Reply:
That's what's being used in RESTEasy, so I wanted to be consistent

Reply:
Mkay but sigh :).